api/net/Areas/Admin/Controllers/ReportInstanceController.cs

```diff
@@ -146,7 +146,7 @@ public async Task<IActionResult> Publish(int id)
         var username = User.GetUsername() ?? throw new NotAuthorizedException("Username is missing");
         var user = _userService.FindByUsername(username) ?? throw new NotAuthorizedException($"User [{username}] does not exist");
 
-        instance.Status = ReportStatus.Submitted;
+        instance.Status = instance.Status == Entities.ReportStatus.Pending ? Entities.ReportStatus.Submitted : instance.Status;
```

Comment:
Don't change the status here unless it was pending.  Provides better outcome when resending.